docs/plugins/authors.asciidoc

```diff
@@ -112,3 +118,15 @@ AccessController.doPrivileged(
 
 See http://www.oracle.com/technetwork/java/seccodeguide-139067.html[Secure Coding Guidelines for Java SE]
 for more information.
+
+[float]
+=== Meta Plugin
+
+It is also possible to bundle multiple plugins into a meta plugin.
+A directory for each sub-plugin must be contained in a directory called `elasticsearch.
+The meta plugin must also contain a file called `meta-plugin-descriptor.properties` in the folder named
+`elasticsearch`. The format for this file is described in detail here:
+
+https://github.com/elastic/elasticsearch/blob/master/buildSrc/src/main/resources/plugin-descriptor.properties[`/buildSrc/src/main/resources/meta-plugin-descriptor.properties`].
```

Comment:
I'm hesitant to add this link. It links to master, but this will be incorrect (as it could be different) once we backport to 6.x.

Reply:
Why would it be incorrect after the backport ?  We do the same for the `plugin-descriptor.properties` of a normal plugin.

Reply:
It must be changed on the backport (and every time we branch) to be correct. For example, if we add any properties to the file in master, but the docs for 6.2 reference the master file, it would cause an error when loading (unknown property). I didn't know we do the same for the regular plugin descriptor...that is unfortunate. This is broken IMO.

Reply:
I pushed https://github.com/elastic/elasticsearch/pull/28022/commits/50435000b5cc437ca5996d04246e22bfa6afae28 to print the properties file in the docs instead of linking to a specific file. WDYT ?

Reply:
This is much better!